openapi.yaml

```diff
@@ -5593,6 +5589,20 @@ components:
       description: >
         A [shipping carrier](https://www.shipengine.com/docs/carriers/setup/),
         such as `fedex`, `dhl_express`, `stamps_com`, etc.
+    lat:
```

Comment:
Why are not using here `$ref: '#/components/schemas/lat'`?

Reply:
this one is 'components/schemas/lat'